frontend/javascripts/oxalis/controller/combinations/tool_controls.ts

```diff
@@ -726,6 +728,19 @@ export class QuickSelectTool {
 
         quickSelectGeometry.setCoordinates(startPos, currentPos);
       },
+      leftClick: (pos: Point2, _plane: OrthoView, _event: MouseEvent, _isTouch: boolean) => {
+        const state = Store.getState();
+        const startPos = V3.floor(calculateGlobalPos(state, pos));
```

Comment:
This is a local variable correct? not the `startPos` part of the parent method `getPlaneMouseControls`. Thus, I'd like to improve the naming here as the clicked position is not a "startPos" but more something like "clickedPos" or "clickedPosGlobal" or "globalclickedPos"

```suggestion
        const clickedPos = V3.floor(calculateGlobalPos(state, pos));
```